### DISCUSSION TITLE
[Feature Request] Integrate a custom card for the dashboard

### DISCUSSION THREAD

**Discussion by storm1er** [alandtse/tesla#651] [Ideas]
**Is your feature request related to a problem? Please describe.**
Creating a custom card for the dashboard can be hard, I'm just learning home assistant and yay, I can do it, but it could be a lot easier.

**Describe the solution you'd like**
Well, this? https://forum.hacf.fr/t/lintegration-tesla-et-automatisation/7572/104

**Describe alternatives you've considered**
I'm currently using the link above, with some UX issues like psi to bar & miles to km conversion, etc

**Comment by alandtse** [comment]
I'm not sure integrations can define any default UI. Perhaps someone can post a template in the wiki. Moving this to discussion.
